File: Python/tests/test_oms_5bias.py
import pytest

from hbayesdm.models import oms_5bias


def test_oms_5bias():
    _ = oms_5bias(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_oms_5bias.R
context("Test oms_5bias")
library(hBayesDM)

test_that("Test oms_5bias", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(oms_5bias(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
